scripts/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+cd ${OKTA_HOME}/${REPO}
+
+setup_service grunt
+setup_service bundler
+
+# Use newer, faster npm
+npm install -g npm@4.0.2
+
+# Install required dependencies
+npm install -g @okta/ci-update-package
+npm install -g @okta/ci-pkginfo
+
+if ! bundle install; then
+  echo "bundle install failed! Exiting..."
+  exit ${FAILED_SETUP}
+fi
+
+if ! npm install --no-optional --unsafe-perm; then
```

Comment:
The no-optional is necessary due to fsevents. --unsafe-perm is necessary due to new CI running as the root user.